test/e2etest.py

```diff
@@ -876,15 +876,17 @@ def test_post_plan(sample_user, sample_team, sample_template_name):
                     "priority": "low",
                     "wait": 600,
                     "repeat": 0,
-                    "template": sample_template_name
+                    "template": sample_template_name,
+                    "optional": 0
```

Comment:
Make sure there's tests with optional undefined to ensure backwards-compatibility

Reply:
Updated the tests to have some calls with no inclusion of the optional field.